src/view/dataElements/xdmObject/helpers/getInitialFormState.js

```diff
@@ -64,6 +64,7 @@ const generateNodeId = () => {
  * and is made available for convenient access so it doesn't have to
  * be repeatedly computed on every render.
  *
+ * @param {Object} sandboxMeta The sandbox meta object.
```

Comment:
It's a little bit unclear to me why this is called "meta".  It looks like you were following a naming pattern that Aaron had setup here. Is it just "Details about the selected sandbox"?

Reply:
Yes